x-pack/test/screenshot_creation/apps/response_ops_docs/stack_connectors/ibm_resilient_connector.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  getExternalServiceSimulatorPath,
+  ExternalServiceSimulator,
+} from '@kbn/actions-simulators-plugin/server/plugin';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const actions = getService('actions');
+  const commonScreenshots = getService('commonScreenshots');
+  const screenshotDirectories = ['response_ops_docs', 'stack_connectors'];
+  const pageObjects = getPageObjects(['common', 'header']);
+  const testSubjects = getService('testSubjects');
+  const kibanaServer = getService('kibanaServer');
+
+  describe('ibm resilient connector', function () {
+    let resilientSimulatorURL: string = '<could not determine kibana url>';
+
+    before(async () => {
+      resilientSimulatorURL = kibanaServer.resolveUrl(
+        getExternalServiceSimulatorPath(ExternalServiceSimulator.RESILIENT)
+      );
+    });
+
+    beforeEach(async () => {
+      await pageObjects.common.navigateToApp('connectors');
+      await pageObjects.header.waitUntilLoadingHasFinished();
+    });
+
+    it('ibm resilient connector creation screenshots', async () => {
+      await pageObjects.common.navigateToApp('connectors');
+      await pageObjects.header.waitUntilLoadingHasFinished();
+      await actions.common.openNewConnectorForm('resilient');
+      await testSubjects.setValue('nameInput', 'IBM Resilient test connector');
+      // await testSubjects.setValue('config.apiUrl-input', 'https://example.com');
```

Comment:
If we use "https://example.com", we receive 404 errors trying to retrieve the incident types when clicking the "Save & Test" button. If we use the `resilientSimulatorURL`, however, it's deemed an invalid URL and we cannot click save:

![image](https://github.com/elastic/kibana/assets/26471269/a7c66da1-0676-47be-98dd-e75b099155ec)